### ISSUE TITLE
Will this work for other Source games like Gmod or tf2?

### ISSUE THREAD

**Issue by ghost** [ericek111/linux-csgo-downloadfixer#6]
I'm having the exact same error elsewhere, and I was wondering if this could be used on games with the same engine?

**Comment by ericek111** [comment]
nope, it has to be modified, but I'll look into it. :) Thanks for your suggestion.
